src/translations/fr.json

```diff
@@ -273,6 +274,123 @@
     "descLfAllCountries": "Tous les pays",
     "descLfCountriesToBalance": "Pays participant à la compensation\u00a0:",
 
+    "DynaFlow_svcRegulationOn": "svcRegulationOn",
```

Comment:
No translation ?

Reply:
translations are not managed in this PR. I will remove them